### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by carlzoo** [carlzoo/scalperr-ui#37]
The frontend should be meet accessibility requirements, which is the law in some countries.

A guide to follow: https://reactjs.org/docs/accessibility.html
